### ISSUE TITLE
Delay in drawing when view controller was pushed

### ISSUE THREAD

**Issue by Luke47** [jessesquires/JSQMessagesViewController#2010]
## New issue checklist

- [x] I have read the [`README`](https://github.com/jessesquires/JSQMessagesViewController/blob/develop/README.md), [documentation](http://cocoadocs.org/docsets/JSQMessagesViewController/), and [FAQ](https://github.com/jessesquires/JSQMessagesViewController/blob/develop/Documentation/faq.md).
- [x] [Contributing guidelines](https://github.com/jessesquires/JSQMessagesViewController/blob/develop/.github/CONTRIBUTING.md) confirmation: ____
- [x] I have searched [existing issues](https://github.com/jessesquires/JSQMessagesViewController/issues?q=is%3Aissue+sort%3Acreated-desc) and **this is not a duplicate**.

## General information

- `JSQMessagesViewController` version: Latest from develop
- iOS version: 10.2
- Devices/Simulators: Simulator (iPhone 7)
- Reproducible in the demo project? (Yes/No): Yes
- Any related issues: nothing found

## What happened?

I've pulled the latest version from `develop` and whenever I push the JSQMessagesViewController, the transition animation shows and a new view slides in from the right (as it should be). However, at first this new view is completely blank / white. Only after a second or so it shows the navigationbar on top and the input toolbar at bottom (and the messages ofc)

This is also the case in the demo project.

What could be the cause and how to fix this? My app which uses an older version of this library (Pulled in Oct 2015) runs perfectly fine with iOS 10.2.

**Comment by chlebta** [comment]
I've the same issue, when pushing VC the navigation bar take time to be drawn!

**Comment by Cykey** [comment]
I have the same issue.

@jessesquires Through `git bisect`, I have found that the problematic commit is 8928144d581b1b46e6632a093c96182519845112.

Thanks!

**Comment by Cykey** [comment]
@jessesquires More specifically, [this code](https://github.com/jessesquires/JSQMessagesViewController/blob/8928144d581b1b46e6632a093c96182519845112/JSQMessagesViewController/Controllers/JSQMessagesViewController.m#L668-L671).

Changing the return value to `NO` makes this issue go away, but has the side-effect of preventing the `UIToolbar` from appearing. :(

Any help would be appreciated.
Thanks!

**Comment by Cykey** [comment]
Okay, so it seems like this is a UIKit issue. 😢 
I have filled a [bug report](https://openradar.appspot.com/radar?id=6119674303479808).
Thanks!

**Comment by jessesquires** [comment]
Hello everyone!

I'm sorry to inform the community that I'm officially deprecating this project. 😢 Please read my blog post for details:

http://www.jessesquires.com/blog/officially-deprecating-jsqmessagesviewcontroller/

Thus, I'm closing all issues and pull requests and making the necessary updates to formally deprecate the library. I'm sorry if this is unexpected or disappointing. Please know that this was an extremely difficult decision to make. I'd like to thank everyone here for contributing and making this project so great. It was a fun 4 years. 😊 

Thanks for understanding,
— jsq
